apps/web/src/hooks/useAddWalletModal.tsx

```diff
@@ -59,18 +67,43 @@ export default function useAddWalletModal() {
         content: (
           <AddWalletModal queryRef={query} onConnectWalletSuccess={onConnectWalletSuccess} />
         ),
-        headerText: 'Connect your wallet',
+        headerText: 'Sign in or create an account',
       });
     },
     [query, showModal]
   );
 }
 
-const Container = styled.div`
-  display: flex;
-  justify-content: center;
-  align-items: center;
-
+const Container = styled(HStack).attrs({
+  justify: 'center',
```

Comment:
I think you can pass in justify as a prop directly to Container where it's used justify="center"